package.json

```diff
@@ -27,10 +27,10 @@
     "prismarine-block": "^1.17.0",
     "prismarine-chat": "^1.7.1",
     "prismarine-chunk": "^1.34.0",
-    "prismarine-entity": "^2.2.0",
+    "prismarine-entity": "^2.3.0",
```

Comment:
Oh, and this bump adds a bunch of deprecation warnings, so we might not want to bump prismarine entity without addressing that

Reply:
https://github.com/PrismarineJS/mineflayer/pull/3168